docs/authentication/session-management-console.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Admin Console actions to support the various development options
+id: session-management-console
+description: ''
+slug: /session-management-console
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/15/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+### Token Format
+
+There are two token formats that can be validated: `self-contained` or `referential`.
+
+The main distinguishing factor is whether introspection is optional or required. Self-contained tokens contain claims which can be used to validate the token. Referential tokens require an introspect call in order to retrieve the claims and validate the token.
+
+When validating self-contained tokens, consider if you want to revoke your tokens. If revoked tokens are accepted (as in a MVP case), then the token can be validated [offline](/docs/next/validate-access-tokens#offline-validation) by validating the signature and parsing the claims of the JWT token. In all other cases the token should be [introspected](https://developer.beyondidentity.com/docs/v1/workflows/api-token#introspection-online-validation).
+
+### Token Configuration: Expiration
+
+This is the time, in seconds, for which the token will be considered after minting. The default value is `86400`, or 24 hours.
+
+### Token Configuration: Subject
+
+Property of a principal which is used to fill the subject of a token issued for this application. The default value is `id`. You may also choose to use `email` or `username`.
+
+### Token Configuration: Token Signing Algorithm
+
+The Token Signing Algorithm is is the algorithm to used to sign an application token. The only allowable value at present is `RS256`. RS256 algorithm is an asymmetric algorithm that uses a private key to sign a JWT and a public key to verify that signature.
```

Comment:
might be useful to have an image of console here

Reply:
Added an image of the default options 👍